### PR TITLE
Update logging.yaml documentation

### PR DESCRIPTION
**Pull request by ema**
The current logging.yaml documentation still describes the old Lua
format in various parts. Update it to reflect the YAML format.

### CODE REVIEW

File: doc/admin-guide/files/logging.yaml.en.rst [hunk 1]
```diff
@@ -381,16 +369,18 @@ to be logged:
 
    filters:
    - name: refreshhitfilter
-     accept: pssc MATCH REFRESH_HIT
+     action: accept
+     condition: pssc MATCH REFRESH_HIT
 
-The following is an example of a filter that will cause the value of the first
-query parameter named ``passwd`` to be wiped.
+.. XXX: this fails, wipe is not a valid attribute.
```
Comment (ema) [inline doc/admin-guide/files/logging.yaml.en.rst]: I did not manage to create a wipe filter yet, unfortunately!

**Comment by SolidWallOfCode** [comment]
[approve ci]
[approve ci docs]

**Comment by mlibbey** [comment]
Looks good. Think we want the commits squashed into one.

**Comment by randall** [comment]
[approve ci]

**Comment by zwoop** [comment]
Cherry-picked to 8.0.x

**Pull request merged**